packages/action-menu/README.md

```diff
@@ -272,13 +272,26 @@ When `selects` is set to `single`, the `<sp-action-menu>` element will maintain
 </sp-action-menu>
 ```
 
-## Accessibility
+## Force Popover on Mobile Devices
 
-An `<sp-action-menu>` parent will ensure that the internal `<sp-menu>` features a role of `listbox` and contains children with the role `option`. Upon focusing the `<sp-action-menu>` using `ArrowDown` will also open the menu while throwing focus into first selected (or unselected when none are selected) menu item to assist in selecting of a new value.
+When on a mobile device, `<sp-action-menu>` elements will render an `<sp-tray>` to the bottom of the viewport by default. To override this functionality so that the picker remains an `<sp-popover>` in mobile, you can simply add the attribute `forcePopover` to the `<sp-action-menu>`. To see this functionality in action, load this page from your mobile device or use Chrome DevTools (or equivalent) and select a mobile device once the Device Toolbar (the phone/tablet icon) is active.
```

Comment:
This is a bit contradicting to what the API contract is all about. We are not overriding functionality but rather we are giving the users to choose the visual ui of popover in mobile devices. Rather I would say now there is an option to switch between different modes of popover UI in mobile. Tray or Popover.
Let me know your thoughts!

Reply:
This goes in hand with my comment above! 👍